website/content/docs/agent/telemetry.mdx

```diff
@@ -542,6 +542,8 @@ These metrics are used to monitor the health of the Consul servers.
 | `consul.xds.server.streams`                         | Measures the number of active xDS streams handled by the server split by protocol version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | streams                           | gauge   |
+| `consul.xds.server.idealSessionsMax`                | The ideal maximum number of xDS sessions per server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | sessions                          | gauge   |
+| `consul.xds.server.sessionDrained`                  | Counts the number of xDS sessions that are drained/terminated to rebalance load between servers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | sessions                          | counter |
```

Comment:
1. Yeah sorry, that's a bad/unfortunate habit of mine 😉 let's commit to the word "drain" (and drop "terminated") because it's more common to speak of "connection draining" when talking about load balancers.
2. That's a really good point. I meant to say that "sessions are drained in order to rebalance load between servers" but I think "when rebalancing" is clearer 👍🏻